### DISCUSSION TITLE
SCT version

### DISCUSSION THREAD

**Discussion by tjg1** [opensafely/documentation#535] [Codelists]
Have I missed which version SNOMED CT you are using?

A colleague pointed they couldn't find this, and realised I couldn't either.

Thanks!

Tom

**Comment by StevenMaude** [comment]
According to [this Opencodelists issue](https://github.com/opensafely-core/opencodelists/issues/999), it's the UK version.

**Reply by tjg1** [reply]
Cheers, and which release?

**Reply by brianmackenna** [reply]
We use the UK version 32.2.0 and try to keep it up to date with the latest one. To be clear this is for https://www.opencodelists.org/

The patient facing systems from which the data we generated is used may be using different versions.
